doc/benchmark.dtd

```diff
@@ -15,6 +15,7 @@
 
 <!ELEMENT option (#PCDATA)>
 <!ELEMENT propertyfile (#PCDATA)>
+<!ELEMENT multiproperty (#PCDATA)>
```

Comment:
Default value of `kind` attribute was set to `composite` (check all properties as a single composite property) and corresponding check for that attribute was added in 88bfeae.